lightning/restore/restore.go

```diff
@@ -1246,7 +1247,7 @@ func (rc *RestoreController) cleanCheckpoints(ctx context.Context) error {
 		err = rc.checkpointsDB.RemoveCheckpoint(ctx, "all")
 	}
 	task.End(zap.ErrorLevel, err)
-	return errors.Trace(err)
+	return errors.Trace(errors.Annotate(err, "clean checkpoints"))
```

Comment:
```suggestion
	return errors.Annotate(err, "clean checkpoints")
```

annotate already implies trace.